docs/hazmat/primitives/asymmetric/ec.rst

```diff
@@ -122,6 +122,43 @@ Elliptic Curve Signature Algorithms
         :returns: A new instance of a :class:`EllipticCurvePublicKey`
             provider.
 
+Elliptic Curve Key Exchange algorithm
+-------------------------------------
+
+.. class:: ECDH(algorithm)
```

Comment:
NVM this is not possible as we'd need to passa  bunch of parameters for the KDF too ..
I guess then that shared_key = private_key.exchange(peer_public_key, ECDH(KDF(a, b, c))) is also a sensible interface.
I would put the peer key first.